master/internal/db/postgres_trial.go

```diff
@@ -168,7 +172,7 @@ func (db *PgDB) UpdateTrialRunnerState(id int, state string) error {
 // UpdateTrialRunnerMetadata updates a trial's metadata about its runner.
 func (db *PgDB) UpdateTrialRunnerMetadata(id int, md *trialv1.TrialRunnerMetadata) error {
 	if _, err := db.sql.Exec(`
-UPDATE trials
+UPDATE runs
```

Comment:
we could use a better strategy for updates than this, lol (500 different functions for each field).

Reply:
```
// UpdateTrial updates an existing trial. Fields that are nil or zero are not
// updated.  end_time is set if the trial moves to a terminal state.
func (db *PgDB) UpdateTrial(id int, newState model.State) error {
```

I agree, created https://hpe-aiatscale.atlassian.net/browse/DET-9993